frontend/src/util.js

```diff
@@ -0,0 +1,17 @@
+
+// eslint-disable-next-line no-unused-vars
+export function checkStatus(response) {
+  if (response.status >= 200 && response.status < 300) {
+    return response;
+  }
+  const error = new Error(`HTTP Error ${response.statusText}`);
+  error.status = response.statusText;
+  error.response = response;
+  console.error(error); // eslint-disable-line no-console
+  throw error;
+}
+
+// eslint-disable-next-line no-unused-vars
+export function parseJSON(response) {
```

Comment:
Do we need this function really ?

Reply:
no. I need to rewrite the uploader fetch requests in the same style as the dashboard.
Good catch 👍